### ISSUE TITLE
Give packager a wheels directory instead of vendoring wheel contents

### ISSUE THREAD

**Issue by kislyuk** [aws/chalice#493]
It would be nice if instead of unpacking my wheels into the `vendor` directory, I had the option to point the packager to a directory with wheels, so it ingests them in `_download_dependencies()` before trying any other methods.

https://github.com/aws/chalice/blob/1cdebd90d66435c26958687a1d6a3670eeb06293/chalice/deploy/packager.py#L344-L411

**Comment by kyleknap** [comment]
Yep. Seems reasonable. Marking as a feature request. @stealthycoin What's your thoughts on this?

**Comment by stealthycoin** [comment]
Seems reasonable and could save a lot of time during initial packaging. Only question is how to expose it. Right now Chalice uses folders and files with specific names to control a lot of deployment behavior so this could be as simple as adding another top level directory called `wheels` and anything in there is used as a priority during initial packaging. Vendor would still copy over anything at the end which is a little strange.

**Comment by jamesls** [comment]
FWIW I created https://github.com/aws/chalice/issues/839 to flesh out your proposal of just supporting a `wheels/` directory.
